Update docs

ethanwillis opened the pull request.

ethanwillis committed:
Added function for adding ABI upgrading functionality.

Takes in compiler version of original ABI. Upgrades ABI to latest Solidity standard.

Route is at /update_abi. See code for input/return spec.

se3000 commented:
So much better. One little thing, could you squash that last commit into the one before it?

Thanks!

ethanwillis committed:
Updateeadme
Added a link to where the default webapp will run.

Added some quick api docs to the readme for the various endpoints.

Added a link to the official webdocs.
Added a link to where the default webapp will run.

Added some quick api docs to the readme for the various endpoints.

Added a link to the official webdocs.

ethanwillis commented:
Done.

se3000 commented:
Thanks!
